job_runner/reporting/event.py

```diff
@@ -78,6 +85,10 @@ def report(self, msg):
     def _dump_event(self, event):
         self._event_queue.put(event)
 
+    def _dump_cloud_event(self, attributes: Dict[str, str], data: Any = None):
+        logger.debug("Dump cloud event")
```

Comment:
This is ambiguous… Move it to `_dump_event` and change the line to

    logger.debug("schedule {type(event)} \"{event['type']}\" for delivery") 

or something?